modules/persistent/arrow-persistent-data-structures/gradle.properties

```diff
@@ -0,0 +1,4 @@
+# Maven publishing configuration
+POM_NAME=Arrow Persistent Data Structures
+POM_ARTIFACT_ID=arrow-persistent-data-structures
```

Comment:
Can we look for a cool artifact ID, maybe shorter than this :)

Reply:
Do you have a suggestion? `arrow-persistent-data` maybe?

Reply:
@i-walker ping

Reply:
arrow-persist? arrow-eternal? :D